mailing_list/constants.py

```diff
@@ -0,0 +1,39 @@
+# we only want boost devel for now, leaving the others in case that changes.
+ML_STATS_URLS = [
+    "https://lists.boost.org/Archives/boost/{:04}/{:02}/author.php",
+    # "https://lists.boost.org/boost-users/{:04}/{:02}/author.php",
+    # "https://lists.boost.org/boost-announce/{:04}/{:02}/author.php",
+]
+arg_date_regex = r"^([0-9]+)(?:$|(?:-|/)([0-9]+)(?:$|(?:-|/)([0-9]+)$))"
+author_pattern_regex = r"<li><strong>(.*)</strong>"
+date_pattern_regex = r".*<em>\((\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2})\)</em>"
+
+latin_1_equivs = {
+    8364: 128,
```

Comment:
Could we add a comment explaining why this is needed? I assume it's because there are some non-unicode characters in mailing list content?